src/views/Dashboard/Lab/Account/index.vue

```diff
@@ -200,24 +200,24 @@ export default {
     
     rules(){
       return [
-        file => !file || file.size <= 3_097_152 || 'Document size should be less than 3 MB!',
+        file => !file || file.size <= 3_097_152 || 'The total file size uploaded exceeds the maximum file size allowed (3MB)',
         file => !file || file.type == 'image/jpg' || file.type == 'image/jpeg' || 'Document type should be image/jpg',
```

Comment:
should also mention 'jpeg' in the error message. also, png format should also supported